common/protob/messages-aptos.proto

```
@@ -66,6 +66,16 @@ message AptosSignMessage {
     }
 }
 
+/**
+ * Request: ask device to sign SIWA message
+ * @start
+ * @next AptosMessageSignature
+ */
+ message AptosSignSIWAMessage {
+    repeated uint32 address_n = 1;
+    required string siwa_payload = 2;
```

Buf (1.55.1)

Field named "siwa_payload" should not be required.

(FIELD_NOT_REQUIRED)